web/css/style.css

```
@@ -3,6 +3,11 @@ body {
   font-family: Roboto;
 }
 
+grid {
+  display: grid;
+  grid-template-columns: 1fr 1fr 1fr 1fr;
```

Can also be written as:
```suggestion
  grid-template-columns: repeat(4, 1fr);
```